.npmignore

```diff
@@ -0,0 +1 @@
+__tests__
```

Comment:
So apparently the presence of `.npmignore` overrides `.gitignore`, so if there's anything else (e.g. `__coverage__`) that you don't want to publish to npm you should add it here too.